src/main/resources/static/views/login/login.html

```diff
@@ -1,89 +1,107 @@
-<div class="container">
-    <div class="row">
-        <div class="col-md-6 col-md-offset-3">
-            <div class="panel panel-login">
-                <div class="panel-heading">
-                    <div class="row">
-                        <div class="col-xs-6">
-                            <a href="#" class="active" id="login-form-link">Login</a>
-                        </div>
-                        <div class="col-xs-6">
-                            <a href="#" id="register-form-link">Register</a>
-                        </div>
-                    </div>
-                    <hr>
-                </div>
-                <div class="panel-body">
-                    <div class="row">
-                        <div class="col-lg-12">
-                            <form id="login-form" action="#" method="post" role="form" style="display: block;">
-                                <div class="form-group">
-                                    <input type="text" name="username" id="username" tabindex="1" class="form-control" placeholder="Email" value="">
-                                </div>
-                                <div class="form-group">
-                                    <input type="password" name="password" id="password" tabindex="2" class="form-control" placeholder="Password">
+<body>
+<div class="container-fluid">
+    <div class="row content">
+        <div class="col-sm-3 sidenav hidden-xs">
+            <h2><a style="color: black; text-decoration: none" href="/">Capstone System</a></h2>
+            <ul class="nav nav-pills nav-stacked">
+                <li><a href="/projects">Projects</a></li>
+                <li class="active"><a href="/login">Login</a></li>
+                <li><a href="/">Announcements</a></li>
+            </ul><br>
+        </div>
+        <br>
+        <div class="col-sm-9">
+            <h1 class="portalheader">
+                <i class="icon-login" aria-hidden="true"></i> Login
+            </h1>
+            <div class="row">
+                <div class="col-md-6 col-md-offset-3">
+                    <div class="panel panel-login">
+                        <div class="panel-heading">
+                            <div class="row">
+                                <div class="col-xs-6">
+                                    <a href="#" class="active" id="login-form-link">Login</a>
                                 </div>
-                                <div class="form-group text-center">
-                                    <input type="checkbox" tabindex="3" class="" name="remember" id="remember">
-                                    <label for="remember"> Remember Me</label>
+                                <div class="col-xs-6">
+                                    <a href="#" id="register-form-link">Register</a>

```

Comment:
is it possible we use an ng-click on the div to navigate to the register form. Currently it only works when i select the text only. Same thing goes for the login link.

Reply:
Usually you must select the text for the event to trigger, same goes for the icons on the home page. That was my intention.